spec/lib/rufo/formatter_source_specs/trailing_commas.rb.spec

```diff
@@ -364,13 +388,16 @@ foo(
 
 #~# EXPECTED
 
-[1,
- 2, 3,
- 4 # foo
+[
+  1,
+  2,
+  3,
+  4, # foo
 ]
 
 #~# ORIGINAL
 #~# trailing_commas: true
+#~# print_width: 5
```

Comment:
Is it perhaps worth testing higher print widths _don't_ break onto newline?:
```ruby
#~# ORIGINAL
#~# trailing_commas: true
#~# print_width: 80
[1, 2,    3, 4]

#~# EXPECTED
[1, 2, 3, 4]
```

Reply:
I have added two specs which demonstrate the behavior: https://github.com/ruby-formatter/rufo/blob/add-basic-line-length-based-formatting-support/spec/lib/rufo/formatter_source_specs/method_calls.rb.spec#L722

Of note is that anything prior to the array is not included in the line length calculation.

For example: `func([1, 2, 3])` should have a maximum line length for the array of `print_width - "func(".length` instead the only thing prior to the start of the array that is considered is the indentation. So if it is indented one level then we have `80 - 2` as the print width for the array.

Note that this is only the case while we have not converted the rest of the code base to use this system.